development-guide.md

```diff
@@ -211,6 +211,13 @@ Where SVGs are used, they are used accessibly
 
 ### Images
 
+Decorative images should be implemented using the CSS `background-image` property, so that Assistive Technology can ignore the images. If a decorative image must be implemented using the HTML `<img>` element, you must ensure that:
+ 
+ - The element's `title` attribute is either absent or empty. 
+ - The element's `alt` attribute is empty (`alt="""`). 
```

Comment:
Thanks @AshTNA - what do you think about including `role="presentation"` as an additional bullet point? It's not as well supported as the null `alt=""`, but that situation is likely to improve and it's a more declarative way of indicating (to developers too) that it's a decorative image. I'd also consider including a link to this page: https://www.w3.org/WAI/tutorials/images/decorative/

Reply:
Good idea @gtvj - I have added both of these to the development guide.